eultheme/managers.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+from django.db import models
+from django.db.models import Q
+from django.conf import settings
+from django.utils.timezone import utc
+
+class BannerQuerySet(models.query.QuerySet):
+
+    def active(self):
+        return self.filter(disabled=False, period__enabled=True)
+
+    def first(self):
```

Comment:
But that was introduced in Django 1.6. For 1.5 support, it's needed. I can add a note.